avocado/core/nrunner.py

```diff
@@ -42,12 +42,12 @@
 RUNNERS_REGISTRY_PYTHON_CLASS = {}
 
 
-def check_tasks_requirements(tasks, runners_registry=None):
+def check_runnables_requirements(runnables, runners_registry=None):
```

Comment:
`check_runnables_runner_requirements` seems to be more meaningful as this is responsible for checking just if the runner is available.